src/PerfectGym.AutomergeBot/RepositoryConnection/RepositoryConnectionContext.cs

```diff
@@ -178,12 +178,26 @@ public IReadOnlyList<PullRequest> GetOpenPullRequests()
             var request = new PullRequestRequest
             {
                 State = ItemStateFilter.Open,
+                
             };
             var openPullRequests = client.PullRequest.GetAllForRepository(_repositoryOwner, _repositoryName, request)
                 .Result;
             return openPullRequests;
         }
 
+        public PullRequest GetPullRequest(int pullRequestNumber)
+        {

```

Comment:
Wszystkie metody w tej klasie powinny logować "Debug" z informacją co się bedzie działo.
```suggestion
        {
             _logger.LogDebug("Getting pull request {pullRequestNumber}", pullRequestNumber);
```